components/navbar/navbar.css

```diff
@@ -0,0 +1,37 @@
+.navbar-container{
+    width: 100%;
+    display:flex;
+    flex-direction: row;
```

Comment:
just a suggestion. I you are using same properties more than once like 
display:flex;
align-items:center etc. better put it in a separate class.
But you want to add responsiveness to your page then it is fine.

Reply:
yes i want it responsive when i use it in documentation